docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
+
+ENV GOROOT=/usr/local/go
+
+RUN go get github.com/tools/godep
+RUN go get github.com/hudl/fargo
```

Comment:
Since I'm using GitBash for windows `MSYS_NO_PATHCONV=1` helped. I'll try to make it work as you wanted @seh